torch/distributed/fsdp/fully_sharded_data_parallel.py

```diff
@@ -960,6 +969,7 @@ def __init__(
         self.cpu_offload = cpu_offload or CPUOffload()
         self.backward_prefetch = backward_prefetch
         self.forward_prefetch = forward_prefetch
+        self.all_gather_issue_limit = all_gather_issue_limit
```

Comment:
this mostly should apply to the root FSDP unit, i.e. all FSDP units should share the same limit? what happens with manual wrapping if users specify it differently?